pkg/appliance/functions.go

```diff
@@ -1020,10 +1056,12 @@ func downloadDockerImageBundle(args imageBundleArgs) {
 		args.fileEntryChan <- fileEntry{err: err}
 		return
 	}
+	tmpManifest.Close()
```

Comment:
Do we need a `defer` so the file gets closed even if there is an exception before it gets closed?

Reply:
Actually we need to close it before it's sent on the channel, because it's going to be opened again once it's sent. And it would still not run if there's a panic of some kind even if it's deferred, so it's not guaranteed to be closed anyway.